homeassistant/components/monarch_money/entity.py

```diff
@@ -81,3 +81,9 @@ def available(self) -> bool:
     def account_data(self) -> MonarchAccount:
         """Return the account data."""
         return self.coordinator.data.account_data[self._account_id]
+
+    async def async_get_holdings(self):
+        """Get holdings."""
+        if holdings := self.account_data.holdings:
+            return holdings.to_json()
+        return "{}"
```

Comment:
![image](https://github.com/user-attachments/assets/799346cb-5c38-471c-9eb5-dfcdd183d328)

(From previous PR Discussion)

Reply:
Right, but in theory you can return a ServiceResponse object, rather than a str.

Check Mealie for example